tests/download/test_session.py

```diff
@@ -34,3 +37,46 @@ def test_token_info(session):
 
     for key in ["sub", "aud", "jti", "exp", "name", "email", "sid", "org", "did", "aid", "d"]:
         assert key in info
+
+
+def test_session_with_missing_credentials(fake_token):
+    config_without_credentials = SHConfig()
+    config_without_credentials.reset()
```

Comment:
could again be replaced with `SHConfig(use_defaults=True)` probably

Reply:
Fixed.